### PR TITLE
Update LWC Preview UI strings and Add Preview Service

### PR DESCRIPTION
**Pull request by brandonpage**
### What does this PR do?

- Minor update to strings from CCX UI review.
- Prefix settings with "Preview"
- Move settings and stored device functions to Preview Service.

### What issues does this PR fix or reference?
Addresses code style comments from #2229 

### Functionality Before
Screen Shot 2020-06-05 at 3 50 47 PM

### Functionality After
Screen Shot 2020-06-05 at 2 46 33 PM

### CODE REVIEW

File: packages/salesforcedx-vscode-lwc/src/service/previewService.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    return WorkspaceUtils.getInstance()
+      .getWorkspaceSettings()
+      .get(this.previewOnMobileKey, false);
+  }
+
+  public isRememberedDeviceEnabled(): boolean {
+    return (
+      WorkspaceUtils.getInstance()
+        .getWorkspaceSettings()
+        .get(this.rememberDeviceKey) || false
```
Comment (xyc) [inline packages/salesforcedx-vscode-lwc/src/service/previewService.ts:54]: use the [`defaultValue` param](https://code.visualstudio.com/api/references/vscode-api#1315) for `get` like above?

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ const enum PlatformName {
   ios = 'iOS'
 }
 
-interface PreviewQuickPickItem extends vscode.QuickPickItem {
+export interface PreviewQuickPickItem extends vscode.QuickPickItem {
```
Comment (xyc) [inline packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts:47]: Let's use PlatformName to pass to getRememberedDevice/updateRememberedDevice? `PreviewQuickPickItem` is UI specific. I guess the enum comment is making more sense now :)

**Comment by xyc** [comment]
@lcampos I don't seem to have permissions for viewing license/snyk, security/snyk

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts (hunk 2 above)
Comment (brandonpage) [reply packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts:47]: Good call.  Yes, much cleaner this way.

**Comment by brandonpage** [comment]
@xyc @lcampos Is it possible to have this get merged today?

**Comment by xyc** [comment]
Good to merge from my end

**Pull request merged**